quickjs.h

```diff
@@ -616,6 +616,7 @@ JS_EXTERN JSValue JS_Throw(JSContext *ctx, JSValue obj);
 JS_EXTERN JSValue JS_GetException(JSContext *ctx);
 JS_EXTERN JS_BOOL JS_HasException(JSContext *ctx);
 JS_EXTERN JS_BOOL JS_IsError(JSContext *ctx, JSValue val);
+JS_EXTERN void JS_SetUncatchableError(JSContext *ctx, JSValue val, JS_BOOL flag);
```

Comment:
What use do you have for this API?

Reply:
We use it for preempting the JavaScript execution stack: https://github.com/holepunchto/libqjs/blob/d32c3f5c95dfb4de30c54d86aaee498c6886d914/src/js.c#L5259-L5272

Reply:
Aha, makes sense.